### PR TITLE
Commiting Docker File Changes + Some Other Favicon Shit

### PR DESCRIPTION
**Pull request by rahulmamgain**
## Rebuilds Docker File And Static Content Can Now Be Loaded
- Download Code
- Rebuild Docker Image with 

```
docker build -t conlini/h2h_dev .
```
- Run The Application Using 

```
docker run -d -v "$(pwd)":/usr/src/app -w /usr/src/app -e DB.CHOICE="MYSQL" -e DB.HOST="h2hdev.cbsd9m5mvpho.us-west-2.rds.amazonaws.com" -e DB.PORT="3306" -e DB.USER="h2hdevmaster" -e DB.PASSWORD="h2h#1234" -e DB.NAME="h2h" -p 8081:8000 conlini/h2h_dev
```
- Refer to wiki at

https://tree.taiga.io/project/rahulmamgain-h2h/wiki/docker-setup.
- Please push the built image to dockerhub.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -13,14 +13,10 @@ ENV DB.USER=""
 ENV DB.PASSWORD=""
 ENV DB.CHOICE="SQLLITE"
 
-RUN mkdir -p /usr/src/app
```
Comment (conlini) [inline Dockerfile:16]: Wouldnt doing this make the dockerfile a dev oriented dockerfile?
If not, we would need to have a way to deploy the code in PROD.
One possible way could be to stick a setup.py which would actually build the code as an installable and that gets stuck in the Dockerfile..

File: h2h/settings.py [hunk 2]
```diff
@@ -23,7 +23,7 @@
 SECRET_KEY = os.environ.get("secret.key", "abcd1234")
 
 # SECURITY WARNING: don't run with debug turned on in production!
-DEBUG = False
+DEBUG = True
```
Comment (conlini) [inline h2h/settings.py]: May be we should externalise this as an env var? Let me know

Comment (rahulmamgain) [inline h2h/settings.py]: makes sense

File: Dockerfile (hunk 1 above)
Comment (rahulmamgain) [inline Dockerfile:16]: How will it make it dev oriented

**Pull request merged**